main.py

```diff
@@ -1041,7 +1048,7 @@ def GetPlayGroundGameKey(self, promoData):
         response = None
 
         retryCount = 0
-        while retryCount < 8:
+        while retryCount < 15:
```

Comment:
Why 15?

Reply:
> Why 15?

Selected a more optimal number by experience, the new game on average triggers on 11 attempts + reserve